TEST_APPS/testcases/nanostack_mac_tester/README.md

```diff
@@ -0,0 +1,298 @@
+##Nanostack MAC RF tests
+
+This folder contains nanostack MAC RF tests for Icetea.
+The test located under this folder is dependent of the application [`nanostack_mac_tester`](https://github.com/ARMmbed/mbed-os/blob/master/TEST_APPS/device/nanostack_mac_tester).
+The nanostack MAC tester is disabled by default. To run the test cases with the application, either remove the preprocessor macro from main.cpp or add `-DICETEA_MAC_TESTER_ENABLED` to the `mbed test` command.
+
+Icetea test cases are processed by passing commands through the `mbed-client-cli` command line. It is possible to manually replicate most test cases by following the instructions below.
+
+In test cases with more than one device under test(DUT) the target device is given in the instructions as DUT1, DUT2 or DUT3.
```

Comment:
Parenthetical expressions require a space on either side of them. Please do a global search of this document to make sure there's a space before the opening parenthesis each time.